docs/DeveloperGuide.md

```diff
@@ -115,18 +115,24 @@ How the parsing works:
 * All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
 
 ### Model component
-**API** : [`Model.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/model/Model.java)
+**API** : [`Model.java`](https://github.com/AY2425S1-CS2103-F13-4/tp/blob/master/src/main/java/seedu/address/model/Model.java)
 
-<img src="images/ModelClassDiagram.png" width="450" />
+<img src="images/ModelClassDiagram.png" width="800" />
 
 
 The `Model` component,
 
-* stores the address book data i.e., all `Person` objects (which are contained in a `UniquePersonList` object).
-* stores the currently 'selected' `Person` objects (e.g., results of a search query) as a separate _filtered_ list which is exposed to outsiders as an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
+* stores the address book data i.e., all `Person`, `Job`, and `Company` objects 
+(which are contained in a `UniquePersonList`, `UniqueJobList`, and `UniqueCompanyList` object respectively).
+* stores the currently 'selected' `Person`, `Job`, and `Company` objects (e.g., results of a search query) as a 
+separate _filtered_ list which is exposed to outsiders as an unmodifiable `ObservableList<Person>`, 
+`ObservableList<Job>`, and `ObservableList<Company>` that can be 'observed' e.g. the UI can be bound to this list 
+so that the UI automatically updates when the data in the list change.
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
-* does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
+* does not depend on any of the other three components (as the `Model` represents data entities of the domain, 
+they should make sense on their own without depending on other components)
 
+[//]: # (Not sure what to do with the diagram and text below)
```

Comment:
I would say keep it for now, and maybe nearer to the end we can choose to delete it or not.  

We aren't using this alternative model anyway, and not sure if there is any benefit to switching to it.

Reply:
Alright we should just look at the whole DG near the end of 1.5 and discuss diagrams like this.